src/components/atoms/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react'
+import classes from './styles.module.scss'
+import classNames from 'classnames'
+
+export type Step = {
+  label: string
+  isCurrentStep: boolean
+}
+
+interface ProgressBarProps {
+  steps: Step[]
+}
+
+const ProgressBar: FC<ProgressBarProps> = ({ steps }) => {
+  return (
+    <div className={classes.progressBarContainer}>
```

Comment:
**[blocker]** (Potentially)

Do we need this div around the ul ? Did a quick test here and it seems to be unnecessary